cos-fleet-catalog-connectors-it/aws/aws-it/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+    <parent>
+        <groupId>org.bf2</groupId>
+        <artifactId>cos-fleet-catalog-connectors-it-aws</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>cos-connector-aws-it</artifactId>
+
+    <name>${project.artifactId}</name>
+    <description>${project.artifactId}</description>
+
+    <packaging>jar</packaging>
+
+    <properties>
+        <quarkus.container-image.name>cos-connector-aws-sqs</quarkus.container-image.name>
+        <quarkus.container-image.tag>${cos.connector.version}.${cos.connector.revision}</quarkus.container-image.tag>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>localstack</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-sqs</artifactId>
+            <version>1.12.180</version>
```

Comment:
Maybe it's not important, but since Camel it is using AWS SDK v2, maybe it makes sense to stay at the same level and avoid 1.x

Reply:
yeah I need to switch to v2 but I need to make my own LocalstackContainer as the one from testcontainers has some import from v1 triggers compilation failures